tests/schema/test_pdk.py

```diff
@@ -0,0 +1,99 @@
+import pathlib
+import pprint
+import json
+
+from align.schema.pdk import LayerMetal, LayerVia, LayerViaSet, PDK
+
+my_dir = pathlib.Path(__file__).resolve().parent
+
+
+def test_one():
+    m1 = LayerMetal(
+        name="M1",
+        gds_layer_number=1,
+        direction="v",
+        min_length=1000,
+        min_end_to_end=400,
+        offset=0,
+        width=[600],
+        space=[400],
+        stop_pitch=1000,
+        stop_point=200,
+        stop_offset=0
+    )
+    m2 = LayerMetal(
+        name="M2",
+        gds_layer_number=2,
+        direction="h",
+        min_length=500,
+        min_end_to_end=300,
+        offset=0,
+        width=[400, 500, 500, 600, 600, 500, 500],
+        space=[300, 300, 400, 400, 400, 300, 300],
```

Comment:
@arvuce22  had a PDK with multiple widths whose syntax looks similar but means something completely different. May want to sync and come up with something consistent. This may naturally lead us to multiple metal templates too.

Reply:
Arvind, I believe we agreed on this, please confirm.
One way to handle multiple metal templates (aka metal template instances) is to define a LayerMetalSet just like LayerViaSet. I deferred this to a future PR.